recipes/cmp/all/test_package/test_package.c

```diff
@@ -0,0 +1,10 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdbool.h>
```

Comment:
```suggestion
#include <stdbool.h>
#include <stdint.h>
```

Reply:
Are these included missing from the `"cmp.h"` header?

Reply:
they are not included, correct